src/PostalRegistry.Api.Legacy/PostalInformation/PostalInformationController.cs

```diff
@@ -264,6 +265,92 @@ public class PostalInformationController : ApiController
             return sw.ToString();
         }
 
+        /// <summary>
+        /// Vraag een lijst met wijzigingen van postinfo op, semantisch geannoteerd (Linked Data Event Streams)
+        /// </summary>
+        /// <param name="configuration"></param>
+        /// <param name="context"></param>
+        /// <param name="responseOptions"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+
+        [HttpGet("base")]
```

Comment:
`base` is too generic, like `core`. Is `/v1/postcodes/linked-data` a better alternative?